ilastik/workflows/tracking/conservation/conservationTrackingWorkflow.py

```diff
@@ -353,27 +355,56 @@ def prepare_lane_for_export(self, lane_index):
             withBatchProcessing = True
         )
 
-    def post_process_lane_export(self, lane_index):
+    def post_process_lane_export(self, lane_index, time=1):
```

Comment:
This was my perhaps unexperienced way to make sure, that we check only once for files that could be overwritten, pop up the MessageBox and then don't export. (time=0). For the next round we click the export button, we really want it to export, so time=1. The default parameter is 1, so everything but not 0, in order to ensure writing out even in headless mode.
Perhaps there is a more elegant way?